configure.ac

```diff
@@ -571,6 +571,7 @@ WRAPPER_COMPILER_EXTRA_LDFLAGS=`echo "$WRAPPER_COMPILER_EXTRA_LDFLAGS" | sed -e
 AC_MSG_RESULT([$WRAPPER_COMPILER_EXTRA_LDFLAGS])
 AC_SUBST(WRAPPER_COMPILER_EXTRA_LDFLAGS)
 AC_SUBST(WRAPPER_COMPILER_EXTRA_LIBS)
+AC_SUBST(PTHREAD_CFLAGS)
```

Comment:
Right, what's surprising is that ```AC_SUBST(PTHEAD_CFLAGS)``` does anything at all.  The current usage of ```$PTHREAD_CFLAGS/LIBS``` in ```Makefile.am``` is definitely wrong.  Past Jim had not wanted to include these in ```$CFLAGS``` and ```$LDFLAGS```, but present Jim does not remember why...  I do think I prefer the ```$CFLAGS/$LDFLAGS``` approach to an autoconf substitution (e.g. ```@PTHREAD_CFLAGS```).